### DISCUSSION TITLE
Help with building Android APK (Paid)

### DISCUSSION THREAD

**Discussion by aloraps** [Alex313031/thorium#878] [Q&A]
Hello,

I am looking for help in building an Android APK based off of Thorium V123. I want to use this version for my PWA, so I simply want it to launch Kiosk mode (I believe this is what it is called) where there is no browser UI, and the default homepage is set to my PWA's main page. No other tweaks should be needed, and I only need the build for Android (ideally both 32/64 bit).

If you are interested, I would like to compensate you for your time. Please reach out to me via Discord:
alora.io

Or via email if preferred:
admin@alora.io

Thank you.

**Comment by Alex313031** [comment]
@aloraps Why do you want to use M123 specifically? 

Is your PWA for mobile only? On desktop you could use Electron, and I actually could do that too. We could make it FOSS or non-FOSS.

What is the PWA? Is it related to https://www.alora.io/

**Reply by aloraps** [reply]
> @aloraps Why do you want to use M123 specifically?
> 
> Is your PWA for mobile only? On desktop you could use Electron, and I actually could do that too. We could make it FOSS or non-FOSS.
> 
> What is the PWA? Is it related to https://www.alora.io/


For some reason, garbage collection in Chromium changed after 124 and it makes my application hitch during certain actions that have a lot of short-lived memory created. That is the primary reason for wanting to package it this way rather than just relying on the user's built-in browser as a PWA, we want to ensure the gameplay is smooth.

I could use M124 but there was a bug in Thorium on Android that was causing a crash every time it was opened (except for the first open) which wasn't fixed until M125. So, M123 seemed like the most stable for me on Android.

It is mobile only, so there is no need for a desktop version. As for FOSS/non, either way is fine.

Bonus points (and $) if you are somehow able to squeeze even more performance out of this lol, eg. disabling things that are not needed in the browser since it is only running this PWA. 

Here is the link to the PWA if you'd like to see what we're looking to accomplish: https://mobile.alora.io

**Reply by aloraps** [reply]
@Alex313031 Let me know if still interested, would love to move forward with this :)
